core/i18n/src/ITwinLocalization.ts

```diff
@@ -51,25 +52,35 @@ export class ITwinLocalization implements Localization {
       caches: [],
       ...options?.detectorOptions,
     };
+
     this._initOptions = {
       interpolation: { escapeValue: true },
       fallbackLng: "en",
+      maxRetries: DEFAULT_MAX_RETRIES,
       backend: this._backendOptions,
       detection: this._detectionOptions,
       ...options?.initOptions,
     };
 
     this.i18next
       .use(options?.detectorPlugin ?? i18nextBrowserLanguageDetector)
-      .use(options?.backendPlugin ?? XHR)
+      .use(options?.backendPlugin ?? Backend)
       .use(TranslationLogger);
   }
 
   public async initialize(namespaces: string[]): Promise<void> {
+
+    // Also consider namespaces passed into constructor
+    const initNamespaces: string[] = [this._initOptions.ns || []].flat();
+    const combinedNamespaces: Set<string> = new Set([...namespaces, ...initNamespaces]); // without duplicates
+
+    const defaultNamespace: string = this._initOptions.defaultNS ?? namespaces[0];
```

Comment:
I used `namespaces` instead of `combinedNamespaces` since `namespaces` is relative to the `initialize()` function. I.e., you are calling `initialize()` with a list of namespaces, so I let that local list take precedence over the list passed into the constructor. (And that only matters if no `defaultNS` was set in the constructor.)